### PR TITLE
telescope for ereal and other lemmas

### PR DESCRIPTION
**Pull request by affeldt-aist**
used in PR #371

### CODE REVIEW

**Comment by affeldt-aist** [comment]
you meant `fine` maybe

File: CHANGELOG_UNRELEASED.md [hunk 1]
```diff
@@ -260,7 +262,7 @@
     `mule_pinfty_ninfty`, `mule_pinfty_pinfty`, `mule_ninfty_ninfty`,
     `mule_neq0`, `mule_ge0`, `mule_le0`, `mule_gt0`, `mule_le0_ge0`,
     `mule_ge0_le0`
-
+  + renamed `real_of_extended` to `fine` (definition and lemmas)
```
Comment (proux01) [inline CHANGELOG_UNRELEASED.md:265]: Also `fineN`, `fineD`, `fineK` and `fine_expand`?

Comment (affeldt-aist) [reply CHANGELOG_UNRELEASED.md]: I was about to add a couple of other lemmas so I documented the names change on this occasion.

**Review by CohenCyril — APPROVED** [review]
LGTM

**Pull request merged**